showcase.json

```diff
@@ -3835,15 +3834,16 @@
   {
     "id": "icpexchange",
     "name": "ICPEx",
-    "oneLiner": "Decentralized exchange built entirely on the chain based on the ICP.",
+    "oneLiner": "Full-chain decentralized exchange powered by the Proactive Market Maker (PMM) algorithm.",
     "tags": [
       "DeFi"
     ],
-    "description": "Decentralized exchange built entirely on the chain based on the ICP.",
+    "description": "A full-chain decentralized exchange powered by the Proactive Market Maker (PMM) algorithm and is committed to becoming a decentralized financial hub serving Web3.",
```

Comment:
* Replace _and is_ with either _that is_ or simply a comma.
* Suggest changing _becoming a_ to _becoming the leading_